batchflow/models/tf/encoder_decoder.py

```diff
@@ -20,16 +20,16 @@ class EncoderDecoder(TFModel):
         encoder : dict
             base : TFModel
                 Model implementing ``make_encoder`` method which returns tensors
-                with encoded representation of the inputs.
+                with encoded representation of the inputs. Optional
```

Comment:
move `optional` keyword to the first line of parameter description, e.g. `base : TFModel, optional`, just like [numpy style](https://numpydoc.readthedocs.io/en/latest/format.html) advises